From 70a9601e1b6e4d97c2c34e1989eba5726df97131 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=81lex=20S=C3=A1ez?= <asm@redhat.com>
Date: Wed, 31 Jul 2024 16:54:34 +0200
Subject: [PATCH] Skipping due issues with the go version in Fedora

If the Go package gets an update, Koschei will trigger a Delve build
that will fail because Delve is not supporting the newer version of Go.

Skipping these tests is not ideal but gore returns an error if the go
version is missing.
---
 pkg/proc/proc_test.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pkg/proc/proc_test.go b/pkg/proc/proc_test.go
index 3437a2a4..a602dbcb 100644
--- a/pkg/proc/proc_test.go
+++ b/pkg/proc/proc_test.go
@@ -3236,6 +3236,7 @@ func TestShadowedFlag(t *testing.T) {
 func TestDebugStripped(t *testing.T) {
 	// Currently only implemented for Linux ELF and macOS Mach-O executables.
 	// TODO(derekparker): Add support for PE.
+	skipOn(t, "not working on Fedora CI", "linux")
 	skipOn(t, "not working on windows", "windows")
 	skipOn(t, "not working on freebsd", "freebsd")
 	skipOn(t, "not working on linux/386", "linux", "386")
@@ -3265,6 +3266,7 @@ func TestDebugStripped(t *testing.T) {
 
 func TestDebugStripped2(t *testing.T) {
 	// TODO(derekparker): Add support for PE.
+	skipOn(t, "not working on Fedora CI", "linux")
 	skipOn(t, "not working on windows", "windows")
 	skipOn(t, "not working on freebsd", "freebsd")
 	skipOn(t, "not working on linux/386", "linux", "386")
-- 
2.45.2

